PublishToBlob.proj

```diff
@@ -24,12 +24,23 @@
     <PushToBlobFeed ExpectedFeedUrl="$(ExpectedFeedUrl)"
                     AccountKey="$(AccountKey)"
                     ItemsToPush="@(NuGetPackages)"
-                    Overwrite="$(PublishOverwrite)" />
+                    Overwrite="$(PublishOverwrite)"
+                    ManifestBranch="$(ManifestBranch)"
```

Comment:
How do these `$(Manifest*)` variables make it into the build?  Are they environment variables or does the build definition need to be modified to pass them in?

Reply:
Modify the build definition.  Mainly pass the source branch/hash.